From eba613ff12b0fdacf9a2ebc2897845d4bd05beb8 Mon Sep 17 00:00:00 2001
From: "Tekriwal, Tanuj" <tanuj.tekriwal@intel.corp-partner.google.com>
Date: Wed, 24 Jun 2026 08:53:07 +0000
Subject: [PATCH] Adding new kernel source, configs and disable trusty

This patch will add new kernel 6.18.21 android
kernel along with the required configs to bring-up
celadon android 17.

This will also disable Trusty temporarily.

Tracked-On: NA
Signed-off-by: Tekriwal, Tanuj <tanuj.tekriwal@intel.corp-partner.google.com>
---
 caas/mixins.spec | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/caas/mixins.spec b/caas/mixins.spec
index d333570..0ab173b 100755
--- a/caas/mixins.spec
+++ b/caas/mixins.spec
@@ -7,10 +7,10 @@ mixinsrel: false
 product.mk: device.mk
 
 [groups]
-kernel: gmin64(useprebuilt=false,src_path=kernel/lts2023-chromium, loglevel=7, interactive_governor=false, relative_sleepstates=false, modules_in_bootimg=false, external_modules=,debug_modules=, use_bcmdhd=false, use_iwlwifi=false, extmod_platform=bxt, iwl_defconfig=, cfg_path=config-lts/lts2023-chromium, more_modules=true,lts2023_chromium_src_path=kernel/lts2023-chromium, lts2023_chromium_cfg_path=config-lts/lts2023-chromium, linux_intel_lts2023_src_path=kernel/linux-intel-lts2023, linux_intel_lts2023_cfg_path=config-lts/linux-intel-lts2023,lts2024_android_src_path=kernel/lts2024-android,lts2024_android_cfg_path=config-lts/lts2024-android)
+kernel: gmin64(useprebuilt=false,src_path=kernel/android_ack_6.18/common, loglevel=7, interactive_governor=false, relative_sleepstates=false, modules_in_bootimg=false, external_modules=,debug_modules=, use_bcmdhd=false, use_iwlwifi=false, extmod_platform=bxt, iwl_defconfig=, cfg_path=config-lts/lts2023-chromium, more_modules=true,lts2023_chromium_src_path=kernel/lts2023-chromium, lts2023_chromium_cfg_path=config-lts/lts2023-chromium, linux_intel_lts2023_src_path=kernel/linux-intel-lts2023, linux_intel_lts2023_cfg_path=config-lts/linux-intel-lts2023,lts2024_android_src_path=kernel/lts2024-android,lts2024_android_cfg_path=config-lts/lts2024-android,wcl_android_src_path=kernel/android_ack_6.18/common,wcl_android_cfg_path=config-lts/android_ack_6.18)
 disk-bus: auto
-boot-arch: project-celadon(uefi_arch=x86_64,fastboot=efi,ignore_rsci=true,disable_watchdog=true,watchdog_parameters=10 30,verity_warning=false,txe_bind_root_of_trust=false,bootloader_block_size=4096,verity_mode=false,disk_encryption=false,file_encryption=true,metadata_encryption=true,fsverity=true,target=caas,ignore_not_applicable_reset=true,self_usb_device_mode_protocol=true,usb_storage=true,live_boot=true,userdata_checkpoint=true,data_use_f2fs=true,trusty=true)
-sepolicy: enforcing
+boot-arch: project-celadon(uefi_arch=x86_64,fastboot=efi,ignore_rsci=true,disable_watchdog=true,watchdog_parameters=10 30,verity_warning=false,txe_bind_root_of_trust=false,bootloader_block_size=4096,verity_mode=false,disk_encryption=false,file_encryption=true,metadata_encryption=true,fsverity=true,target=caas,ignore_not_applicable_reset=true,self_usb_device_mode_protocol=true,usb_storage=true,live_boot=true,userdata_checkpoint=true,data_use_f2fs=true,trusty=false)
+sepolicy: permissive
 bluetooth: btusb(ivi=false)
 audio: project-celadon
 vendor-partition: true(partition_size=600,partition_name=vendor)
@@ -52,14 +52,14 @@ lights: true
 power: true(power_throttle=true)
 debug-usb-config: true(source_dev=dvcith-0-msc0)
 intel_prop: true
-tee: trusty(ref_target=celadon_64)
+tee: false
 memtrack: true
-tpm: true
+tpm: false
 avx: auto
 health: hal(ivi=false)
 slot-ab: true
 abota-fw: true
-firststage-mount: true(trusty=true)
+firststage-mount: false
 cpuset: autocores
 usb-init: true
 usb-audio-init: false
-- 
2.34.1

